oap-server/server-fetcher-plugin/prometheus-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/fetcher/prometheus/provider/PrometheusFetcherProvider.java

```diff
@@ -62,8 +67,22 @@
 
     private ScheduledExecutorService ses;
 
+    private final HistogramMetrics histogram;
+
+    private final CounterMetrics errorCounter;
+
     public PrometheusFetcherProvider() {
         config = new PrometheusFetcherConfig();
+        MetricsCreator metricsCreator = getManager().find(TelemetryModule.NAME)
+                .provider()
+                .getService(MetricsCreator.class);
+        histogram = metricsCreator.createHistogramMetric(
+                "metrics_scratch_latency", "The process latency of metrics scratching",
```

Comment:
```suggestion
                "metrics_fetcher_latency", "The process latency of metrics scratching",
```
`scratch` -> `fetcher`.